### DISCUSSION TITLE
클러스터 마커 깜빡거림 영상

### DISCUSSION THREAD

**Discussion by 1119wj** [boostcampwm-2024/web09-DailyRoad#199] [Show and tell]
원래 지도 클러스터 라이브러리를 바로 적용한 후
https://github.com/user-attachments/assets/7dabc6e9-eefa-4177-b331-feae53876816

지도 클러스터 라이브러리 커스텀한 후
https://github.com/user-attachments/assets/cded125f-7ae0-4922-bfef-431fc5ab887d

**Comment by Miensoap** [comment]
문서화 잘 하면 좋겠네요 👍
